cypher/cypher-docs/src/docs/dev/ql/administration/show-databases-columns.asciidoc

```diff
@@ -0,0 +1,18 @@
+[options="header" cols="2l,4,1,2"]
+|===
+|Column |Description|Included by default|Notes
+|name|The name of the database.|{check-mark}|
+|databaseID|The database unique ID.||
+|serverID|The server instance ID.||
+|address|Instance address in a clustered DBMS.
+The default for standalone database is `neo4j://localhost:7687`|{check-mark}|
+|role|the current role of the database (`standalone`, `leader`, `follower`, `read_replica`, `unknown`).|{check-mark}|
+|requestedStatus|The expected status of the database.|{check-mark}|
+|currentStatus|The actual status of the database.|{check-mark}|
+|error|An error message explaining why the database is not in the correct state.|{check-mark}|
+|default|Show if this is the default database for the DBMS.|{check-mark}|Not returned by `SHOW HOME DATABASE` or `SHOW DEFAULT DATABASE`
+|home|Shown if this is the home database for the current user.|{check-mark}|Not returned by `SHOW HOME DATABASE` or `SHOW DEFAULT DATABASE`
+|lastCommittedTxn|The ID of the last transaction received.||
+|replicationLag|Number of transactions the current database is behind compared to the database on the primary instance. The lag is expressed in negative integers. In standalone environments, the value is always 0.||
```

Comment:
Yes, this is a better order. I've raised a product card to change the default to this order, but since we officially do guarantee the order I think this is okay for now.